### PR TITLE
BUG 1775630: fix olm.skipRange version

### PR DESCRIPTION
**Pull request by zshi-redhat**

### CODE REVIEW

File: manifests/4.3/ptp-operator.v4.3.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ metadata:
     createdAt: 2019-10-14
     certified: "false"
     repository: https://github.com/openshift/ptp-operator
-    olm.skipRange: ">=4.3.0 <4.4.0"
+    olm.skipRange: ">=4.3.0 <4.3.0"
```
Comment (s1061123) [inline manifests/4.3/ptp-operator.v4.3.0.clusterserviceversion.yaml]: Does it means skip everything (nothing more than+equal 4.3.0 and less than 4.3.0)?

Comment (zshi-redhat) [reply manifests/4.3/ptp-operator.v4.3.0.clusterserviceversion.yaml]: Yes, it means if the version is in that range, then don't auto upgrade the operator.
In above case, it should skip 4.3.0 version and upgrade to any version that larger than 4.3.0, for example, if there is a z-stream version 4.3.1.

**Pull request merged**